truss/cli.py

```diff
@@ -177,7 +177,25 @@ def run_image(target_directory, build_dir, tag, port, attach):
 @error_handling
 @echo_output
 def predict(target_directory, request, build_dir, tag, port, run_local, request_file):
-    """Runs prediction for a Truss in a docker image or locally"""
+    """
+    Runs prediction for a Truss in a docker image or locally
+    
+    Runs the docker image for a Truss.
```

Comment:
nit: Do we still want those 2 sentences? 
nit: Should `Runs prediction for a Truss in a docker image or locally` end with a `.`?